tests/worker/test_workflow.py

```diff
@@ -4927,3 +4927,72 @@ async def test_workflow_wait_utility(client: Client):
             task_queue=worker.task_queue,
         )
         assert len(result) == 10
+
+
+@workflow.defn
+class CurrentUpdateWorkflow:
+    def __init__(self) -> None:
+        self._pending_get_update_id_tasks: List[asyncio.Task[str]] = []
+
+    @workflow.run
+    async def run(self) -> List[str]:
+        # Confirm no update info
+        assert not workflow.current_update_info()
+
+        # Wait for all tasks to come in, then return the full set
+        await workflow.wait_condition(
+            lambda: len(self._pending_get_update_id_tasks) == 5
+        )
+        assert not workflow.current_update_info()
+        return list(await asyncio.gather(*self._pending_get_update_id_tasks))
+
+    @workflow.update
```

Comment:
Should we also test this is available in the validator?

Reply:
Yes, I will add that to the test.

Reply:
Done